officers.json

```diff
@@ -40,6 +40,48 @@
     "term_end": "SPRING_2019",
     "graduating_semester": "SPRING_2021"
   },
+  {
+    "name": "Jacob Raffe",
+    "position": "SECRETARY",
+    "description": "Please fund us",
+    "github": "jakeraffe",
+    "slack": "binjamin",
+    "personal_site": "",
+    "image": "",
+    "special_ability": "",
+    "join_date": "",
+    "term_start": "SPRING_2019",
+    "term_end": "SPRING_2020",
+    "graduating_semester": "SPRING_2020"
+  },
+  {
+    "name": "Brent Yurek",
+    "position": "SENIOR TECHNICAL_OFFICER",
```

Comment:
* Missing underscore between `SENIOR` and `TECHNICAL`
* Make sure that the new role is also reflected in `server.py`
* Make sure that the new role is also reflected in `frontend-site/src/modules/officers.js` (order of names in this file determines sort order on front end, so be sure to put rank in the correct order)